test/stubs/stubVeiledere.ts

```diff
@@ -0,0 +1,15 @@
+import nock from 'nock';
+import { SYFOVEILEDER_ROOT } from '../../src/utils/apiUrlUtil';
+import aktivEnhetMockData from '../../Mock/Data/aktivEnhet.json';
+import veiledereMockData from '../../Mock/Data/veiledere.json';
+import axios from 'axios';
+
+export const stubVeiledere = () => {
+  axios.defaults.adapter = require('axios/lib/adapters/http');
+
+  nock('http://localhost:80')
```

Comment:
'http://localhost:80' går igjen mange steder og kan med fordel tas ut for gjenbruk.

Reply:
trekker ut i felles basePath